.github/ISSUE_TEMPLATE/3_Question.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Support question
-about: Questions about ARTIQ that are not covered in the documentation (https://m-labs.hk/artiq/manual)
+about: Questions about ARTIQ that are not covered in the documentation (https://m-labs.hk/artiq/manual-master/)
```

Comment:
There's no quick (<5 words) way to explain the difference between them. It's probably best to point them to the most up-to-date information.
The highlighted line is meant to be a short overview of what goes in there, and not ALL possible information.
See it on my fork: [choose issue type](https://github.com/drewrisinger/artiq/issues/new/choose)

Reply:
>  It's probably best to point them to the most up-to-date information.

Unfortunately, there can be API breakage / compatibility issues between master and release branches that people may be using. (Hopefully, it is only a temporary state of affairs that so many people currently use master instead of release-3, due to e.g. Kasli support.)

Reply:
And those API issues are detailed fully in the latest documentation, correct? So pointing them to the latest documentation would give them the most relevant information for their problem.

Whatever, this minor point is not worth my time anymore. Making the change. Approve it whenever you get around to it.

Reply:
Fixed in 7c7e26232ea8ed7681105ef28a9ce0b73b55dd4d

Reply:
> And those API issues are detailed fully in the latest documentation, correct?

They are in the release notes but not in the manual.